### DISCUSSION TITLE
how to send raw serialized data

### DISCUSSION THREAD

**Discussion by mzy2240** [GenieFramework/Genie.jl#672] [Q&A]
let's say I have a dataframe serialized using Arrow, how can I return the arrow table via the REST api?

**Comment by essenciary** [comment]
I'm not very familiar with Arrow files but since it's a binary file, so you would have to stream it. Alternatively, you could extract just the data and serialize it as JSON to send it over the wire, then read it back into an Arrow file at destination.

**Comment by mzy2240** [comment]
thank you for the reply! really appreciate! is there any document on how to stream using Genie?

**Comment by essenciary** [comment]
Unfortunately I don't know of anybody that has done that yet. Of course, it's something that ultimately has to be delegated to the underlying web server (HTTP.jl) so that would be the starting point. It's an interesting problem but not sure we have time to look into it right now. Maybe you can check and follow up, we'll help adding a Genie API.
